src/ov_utils.cpp

```diff
@@ -122,6 +122,11 @@ Status validatePluginConfiguration(const plugin_config_t& pluginConfig, const st
         std::string deviceName;
 
         while (getline(ss, deviceName, deviceDelimiter)) {
+            char bracket = '(';
+            auto bracketPos = deviceName.find(bracket);
+            if (bracketPos != std::string::npos) {
+                deviceName = deviceName.substr(0, bracketPos);
```

Comment:
Why are we doing this? Don't we exclude batching info here?

Reply:
to get physical device and retrieve supported properties, we need to omit content in brackets. For the target device in model compilation, it should be included.